modules/storage-api/src/main/java/org/apache/ignite/internal/storage/MvPartitionStorage.java

```diff
@@ -71,20 +71,39 @@ public interface MvPartitionStorage extends ManuallyCloseable {
     CompletableFuture<Void> flush();
 
     /**
-     * Index of the highest write command applied to the storage. {@code 0} if index is unknown.
+     * Index of the highest write command applied to the storage. {@code 0} if the index is unknown.
      */
     long lastAppliedIndex();
 
     /**
-     * Sets the last applied index value.
+     * Term of the highest write command applied to the storage. {@code 0} if the term is unknown.
```

Comment:
What do you mean by `highest write command`?

Reply:
It's the command that was written with the highest index

Reply:
Can you state that explicitly? It's too confusing otherwise

Reply:
Done. Please take a look whether it's ok now.